pytential/symbolic/mappers.py

```diff
@@ -280,7 +280,12 @@ def map_common_subexpression(self, expr):
 # {{{ FlattenMapper
 
 class FlattenMapper(FlattenMapperBase, IdentityMapper):
-    pass
+    def map_int_g(self, expr):
+        densities, kernel_arguments, changed = rec_int_g_arguments(self, expr)
+        if not changed:
+            return expr
+
+        return replace(expr, densities=densities, kernel_arguments=kernel_arguments)
```

Comment:
Why is this needed? It looks an awful lot like what would be inherited from `IdentityMapper`? (If there is a reason, please add a comment.)

Reply:
Yeah, it is. It didn't work due to https://github.com/inducer/pytential/pull/248#discussion_r1847138517 and I forgot to revert this part.

Removed in https://github.com/inducer/pytential/compare/165d0e140a2538ededcbcf1545b4c21205f27e34..baba46fa9af2222a69b1816438274a443e1baa32